src/styles.css

```diff
@@ -462,4 +462,13 @@ span.tooltip {
   font-size: 0.9rem !important;
 }
 
+.select select {
+  border-radius: 0px;
```

Comment:
On other controls like input, Do we have 0px or 2px?

Reply:
It is 2px and 4px on different pages. Should I change and make all of these consistent?

Reply:
Yes please; make consistent as 2px